packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.controller.js

```diff
@@ -103,7 +103,8 @@ export default class CdnSharedSettingsController {
       },
     ];
     this.hstsMaxAgeValue = this.model.hsts?.config.ttl;
-    [this.hstsMaxAgeUnit] = this.hstsMaxAgeUnits; // Max age is expressed in seconds by default
+    [this.hstsMaxAgeUnit] = this.hstsMaxAgeUnits.slice(-1); // Max age is expressed in months by default
```

Comment:
i believe a _.max(this.hstsMaxAgeUnits) would be cleaner and safer

Reply:
Makes sense @frenautvh . This has been updated. Thanks for the suggestion